ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -196,6 +518,18 @@ let clean_local_resources () : unit =
     error "%s : %s" msg (ExnHelper.string_of_exn e) ;
     raise (Auth_service_error (E_GENERIC, msg))
 
+let domainify_uname ~domain uname =
+  let open Astring.String in
```

Comment:
This looks to me like something that could be more reliably checked with a regular expression. However, I'm fine with it as is.